content/docs/reproduce-locally.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Reproduce CI environment locally"
+date: 2022-03-01
+disableToc: false
+weight: 10
+---
+
+# Reproduce CI environment locally
+
+This used to be a question [in our FAQ]({{< ref
+"faq#a-command-failed-in-packit-service-how-do-i-reproduce-it-locally" >}}) and
+now we have a dedicated document to cover this.
+
+
+## SRPM builds locally
+
+```
+$ packit srpm
+```
+
+Packit will create a SRPM out of the current checkout. Simple and clear.
+
+
+## SRPM builds in Copr
+
+When your SRPM is being built in Copr (because [`srpm_build_deps`]({{< ref
+"configuration.md#srpm_build_deps" >}}) is set in your packit config), this
+section describes how you can reproduce the build procedure locally.
+
+Thankfully, Copr is very transparent how it performs builds. When you open a
+build log (e.g. builder-live.log.gz), you'll see a command at the top which
+Copr invokes to perform a build. Let's do that in a container.
+
+```
+$ sudo podman run --privileged -ti fedora:36 bash
+```
```

Comment:
Reading this once more ... are you sure `sudo` is actually needed?

Reply:
actually yes, I explained it below

if you can give me tips how to run mock in a container, I'd love to improve this line and drop as many privileges as possible

Reply:
Now I understand and you're correct, indeed sudo is not needed; thank you!

https://github.com/packit/packit.dev/pull/408